docs/make_julia_for_numpy_users.jl

```diff
@@ -191,6 +191,19 @@ tensor_create_table = TableMaker.ComparisonTable(
             concatenate columns of a and b
             """
         ),
+        (
+            p="""
+            np.hstack((a, b, ...))  # or
+            np.concatenate((a, b, ...), 1)
+            """, 
+            j="""
+            reduce(hcat, [a, b, ...]) 
+            """, 
+            n="""
+            concatenate columns of many arrays
```

Comment:
This needs a period or parentheses. At the moment the sentence "In Julia [...]  many arrays" is just squashed against the fragment before it.